src/actions/__tests__/action.test.js

```diff
@@ -6,23 +6,23 @@ const mock = new MockAdapter(axios);
 const actions = new Actions();
 
 describe("Actions", () => {
-  it("shold expect #startGameAction to send a request", done => {
+  it("should expect #startGameAction to send a request", done => {
     const response_data = {
       message: "game started successfully",
       game_data: { "1": " " }
     };
-    mock.onPost("https://emmanuel-tic-tac-toe.herokuapp.com/startgame").reply(200, response_data);
+    mock.onPost("http://localhost:9292/startgame").reply(200, response_data);
```

Comment:
It might make sense to make the base URL configurable through the environment so that you won't have to change the test when you deploy the app to a different server.